frame/bonded-finance/src/lib.rs

```diff
@@ -376,6 +375,10 @@ pub mod pallet {
 				}
 			})
 		}
+
+		pub(crate) fn account_id(offer_id: T::BondOfferId) -> AccountIdOf<T> {
+			T::PalletId::get().into_sub_account(offer_id)
+		}
```

Comment:
This function is only used inside the crate for "pallet" and "benchmark" purposes. IMO, it does not make sense the leak pallet methods visibility for public usage

Reply:
Good catch, was made public for tests of course.